src/main/java/org/codehaus/plexus/archiver/zip/AbstractZipArchiver.java

```diff
@@ -212,6 +223,14 @@ public boolean isFilesonly()
         return doFilesonly;
     }
 
+    public Long getFixedEntryModificationTime() {
+        return fixedEntryModificationTime;
+    }
+
+    public void setFixedEntryModificationTime(Long fixedEntryModificationTime) {
```

Comment:
I really think this should be some kind of Date object, not a long. I am also contemplating if this setter should be on AbstractArchiver, others could just throw UnsupportedOepratiionException